nsis/src/nsis/NsisAnalyzer.java

```diff
@@ -102,6 +104,29 @@ public boolean added(Program program, AddressSetView set, TaskMonitor monitor, M
     return true;
   }
 
+  /**
+   * Returns the Operation object associated with the given instruction. The association is made
+   * with the opcode of the instruction.
+   * 
+   * @param instruction to create the Operation object with
+   * @return the Operation object associated to the instruction
+   * @throws MemoryAccessException
+   */
+  private Operation toOperationType(Instruction instr) throws MemoryAccessException {
```

Comment:
Suggestion: use name `toOperation`, as the return type name is `Operation`, not `OperationType`.